### ISSUE TITLE
Translation platform

### ISSUE THREAD

**Issue by aha999** [RushikeshKamewar/PrivacyDashboard#3]
It would be great if there was a way for general public to contribute translations.

I can recommend [Weblate](https://weblate.org/), as the platform itself is open source (compared to others), can be self hostable, and for open source projects like yours, they host translations for free. 

Weblate can automatically make pull requests when new content is translated and if you want you can have various widgets shown on github, that display how much percentage each applied language is translated to.  It's much more simple for people to translate this way and it's free.

- [set up your libre project](https://hosted.weblate.org/hosting/) 

screenshot:
![screenshot](https://user-images.githubusercontent.com/50620416/119900111-57ffdf00-bf44-11eb-95e6-892f025a7627.png)

**Comment by woutiscoding** [comment]
What about crowdin? I only used it twice so no real experience or knowledge about it.

**Comment by aha999** [comment]
> What about crowdin? I only used it twice so no real experience or knowledge about it.

Yes, there are many platforms... Crowdin, Transifex, Weblate, Pontoon, etc. and i translated on most of the popular platforms, but Only **Weblate is open source**, other are proprietary, and it can translate as well as other platforms, can be self hosted and UI looks better if you ask me. It also has github integration for automatic pull requests when new translations are made, Widgets with translation progress can be put on Github readme, so it's very nice and imo more than most people could need :) 

[Read Here](https://www.mumble.info/blog/switch-to-weblate-as-translation-platform/) how a popular project Mumble switched to Weblate and why.

**Comment by woutiscoding** [comment]
Oh OK! Super

**Comment by woutiscoding** [comment]
I myself use an open-source localization platform which i self-host but the functionality is super-limited

**Comment by aha999** [comment]
> I myself use an open-source localization platform which i self-host but the functionality is super-limited

That's great! which one ? it might help you that open source weblate has self hosting for free if it will help you... https://weblate.org/en/hosting/

**Comment by woutiscoding** [comment]
I don't remember.. Something of awesome-selfhosted that can run in Docker
